tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.php

```diff
@@ -196,7 +196,7 @@ public static function dataSwitchDefault()
                 'testMarker'    => '/* testSimpleSwitchDefaultWithCurlies */',
                 'openerOffset'  => 3,
                 'closerOffset'  => 12,
-                'conditionStop' => 6,
+                'conditionStop' => 3,
```

Comment:
With the above change, this change is no longer needed.

Reply:
Done